test/page_structure.js

```diff
@@ -67,13 +67,13 @@ async function getProperty(page, selector, property) {
             const preloadStylesheet = "preload stylesheet";
 
             assert.equal(await getProperty(page, "link ~ link ~ link", "rel"), preloadStylesheet);
-            assert.ok((await getProperty(page, "link ~ link ~ link", "href")).endsWith("bower_components/primer-css/css/primer.css"));
+            assert.ok((await getProperty(page, "link ~ link ~ link", "href")).startsWith("https://unpkg.com/primer-css@"));
```

Comment:
Let’s be more precise with these assertions. The goal is to ensure we have the right file referenced, so maybe we can do a wildcard for the version number.

Instead of `.startsWith()`,  more or less a regex as such: `.match("https://unpkg.com/primer-css@\d.+/css/primer.css")`